vividus-allure-adaptor/src/main/resources/allure-customization/webjars/js-beautify/1.13.4/styles/stackoverflow-light.min.css

```diff
@@ -0,0 +1,13 @@
+pre code.hljs{display:block;overflow-x:auto;padding:1em}code.hljs{padding:3px 5px}/*!
+  Theme: StackOverflow Light
+  Description: Light theme as used on stackoverflow.com
```

Comment:
why is it needed?

Reply:
<img width="768" alt="image" src="https://github.com/vividus-framework/vividus/assets/30923336/2257dc6d-12b8-48c1-a0f5-c7fa14286168">
is contains styles for highlighting